datacenter/ucp/2.1/guides/admin/configure/external-auth/index.md

```diff
@@ -1,63 +1,77 @@
 ---
-description: Learn how to integrate UCP with an LDAP service, so that you can manage
-  users from a single place.
-keywords: LDAP, authentication, user management
-title: Integrate with LDAP
+description: Learn how to integrate UCP with an LDAP service, so that you can
+  manage users from a single place.
+keywords: LDAP, directory, authentication, user management
+title: Integrate with an LDAP Directory
 ---
 
 Docker UCP integrates with LDAP services, so that you can manage users from a
-single place.
+single place. Many enterprise organizations have an existing directory service
+where people and groups are managed. UCP allows you to sync people and groups
+from your organization's directory to automatically provision users and team
+members for Docker UCP and Docker Trusted Registry.
 
 ## Configure the LDAP integration
 
-To configure UCP to authenticate users using an LDAP service, go to
-the **UCP web UI**, navigate to the **Settings** page, and click the **Auth**
-tab.
+To configure UCP to provision and authenticate users using an LDAP directory,
+go to the **UCP web UI**, navigate to the **Settings** page, and click the
+**Auth** tab.
 
 ![](../../../images/ldap-integration-1.png){: .with-border}
 
-Then configure your LDAP integration.
+Then configure your LDAP directory integration.
 
 **Authentication**
 
 | Field                                            | Description                                                                                                                                                    |
 |:-------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Method                                           | The method used to authenticate users. Managed authentication uses the UCP built-in authentication mechanism. LDAP uses an LDAP service to authenticate users. |
-| Default permission for newly discovered accounts | The permission level assigned by default to a new user. Learn more about default permission levels.                                                            |
+| Method                                           | The method used to provision and authenticate users. The *Managed* method allows an admin to manually provision users which will be able to login with a securely hashed and salted password stored in UCP. The *LDAP* method uses an LDAP directory server to automatically provision users and all logins will be forwarded to the directory server. |
```

Comment:
"which will be able to login with a securely hashed and salted password" technically this is correct but we're mixing two things that might confuse readers.

* Users login with their LDAP credentials
* UCP stores those credentials by salting and hashing the password. If your UCP gets compromised, no one will be able to get the plain text users passwords.